native/chat/chat-thread-background.react.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+import ChatThreadList from './chat-thread-list.react';
+import type { ChatTopTabsNavigationProp } from './chat.react';
```

Comment:
Nit: let's follow the conventions I use in other files for separating imports. The imports should be in four groups:

1. Types first (from any place)
2. Imports from external packages next (like `react`)
3. Imports from other packages in our monorepo next (eg. imports from `lib`)
4. Finally, imports from files in the current package (eg. imports starting with `..` or `.`)